src/Functions/getMacro.cpp

```diff
@@ -27,15 +27,19 @@ class FunctionGetMacro : public IFunction
 {
 private:
     MultiVersion<Macros>::Version macros;
+    bool is_distributed;
```

Comment:
Can't we use serverConstants.cpp ?

Reply:
I tried, but it is not suitable for functions with single argument. 
If we use serverConstants.cpp, we must get the result of `getMacro` in FunctionConstantBase to initialize `constant_value`, which is obvious impossible.